clients/cobol-lsp-vscode-extension/.vscode/launch.json

```diff
@@ -30,6 +30,19 @@
                 "${workspaceFolder}/out/**/*.js"
             ],
             "preLaunchTask": "npm: watch"
+        },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Debug Tests",
+            "program": "${workspaceRoot}\\node_modules\\jest\\bin\\jest.js",
```

Comment:
Is it ok to have them in windows format?

Reply:
do you mean text format or system path?

Reply:
I think I missed it - which is the reason to put the launch under version control?

Reply:
I was thinking for everybody to have it but I will delete it

Reply:
I think it's ok to have it in git if it can be useful for other developers. And I think it's useful.
But can you try this:
```${workspaceRoot}/node_modules/jest/bin/jest.js```

Reply:
well if you added a configuration that is useful for developing/debugging I am not agains :)

Reply:
great, I will try it out